### PR TITLE
Onboarding role selection

### PR DESCRIPTION
**Pull request by lightme-fan**
Hey everyone,

I have done the Onboarding role selection page and containers. Please have a look at it.

Thanks in advance!

Branch: [https://github.com/onja-org/vamalaka-fannar/tree/onboarding-role-selection](https://github.com/onja-org/vamalaka-fannar/tree/onboarding-role-selection)
Clickup: [https://app.clickup.com/t/41k877](https://app.clickup.com/t/41k877)
Refs: #41k877

### CODE REVIEW

File: src/containers/onboardingContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import { FC } from 'react'
```
Comment (andrzej-onja) [inline src/containers/onboardingContainer.tsx:1]: file name should start with capital O

Comment (andrzej-onja) [reply src/containers/onboardingContainer.tsx:1]: Nice I would rename this to be GenericContainerPage so it can be used for any other page.

Comment (andrzej-onja) [reply src/containers/onboardingContainer.tsx:1]: Maybe move it to Pages folder as its more related to Pages although it has container in the name

File: src/containers/roleSelectionContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
+import { Fragment } from 'react'
```
Comment (andrzej-onja) [inline src/containers/roleSelectionContainer.tsx:1]: File name capital R

**Review by mitia-anah — APPROVED** [review]

**Review by Tiaraha-funny — APPROVED** [review]

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**